deploy/operator.yaml

```diff
@@ -27,7 +27,7 @@ spec:
         - --driver=portworx
         - --leader-elect=true
         imagePullPolicy: Always
-        image: openstorage/openstorage-operator:master
+        image: trierra/operator:dev
```

Comment:
same here

Reply:
yes, accidentally committed, thanks for noticing